src/app/features/new-petition/new-petition.component.ts

```diff
@@ -0,0 +1,30 @@
+import { Component, OnInit } from '@angular/core';
+import { ActivatedRoute, Router } from '@angular/router';
+import { IssuePetitionData } from 'src/app/shared/models/exports';
+
+@Component({
+  selector: 'dp-new-petition',
+  templateUrl: './new-petition.component.html',
+})
+export class NewPetitionComponent implements OnInit {
+  protected currentStep: '1' | '21' | '22' | '3' = '1';
```

Comment:
I think it would be better if we use more descriptive wording here. These numbers don't tell too much.